services/reddit/subreddit-subscribers.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const { BaseJsonService, NotFound } = require('..')
+const Joi = require('joi')
+const { optionalNonNegativeInteger } = require('../validators')
+const { metric } = require('../text-formatters')
+
+const schema = Joi.object({
+  data: Joi.object({
+    subscribers: optionalNonNegativeInteger,
+  }).required(),
+}).required()
+
+module.exports = class SubredditSubscribers extends BaseJsonService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'reddit/subreddit-subscribers',
+      pattern: ':subreddit',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Subreddit subscribers',
+        namedParams: { subreddit: 'drums' },
+        staticPreview: {
+          label: 'follow r/drums',
+          message: '77k',
+          color: 'red',
+          style: 'social',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'reddit',
+    }
```

Comment:
This will stick the logo in there by default
```suggestion
      namedLogo: 'reddit',
    }
```